pennylane_lightning_kokkos/src/bindings/Bindings.cpp

```diff
@@ -735,6 +735,8 @@ PYBIND11_MODULE(lightning_kokkos_qubit_ops, // NOLINT: No control over
 
     m.def("kokkos_start", []() { Kokkos::initialize(); });
     m.def("kokkos_end", []() { Kokkos::finalize(); });
+    m.def("kokkos_configuration",
+          []() { Kokkos::print_configuration(std::cout, true); });
```

Comment:
Does this simply write to std::out, or does it return a string? If it simply writes, we need to instead buffer the output of the function to a stream, and return a string to the Python layer.

UPDATE: I looked at the result and think it should be converted to a dictionary with a name for the type and value. This should be query-able by the Python layer to allow us to make decisions based on the values.

Reply:
Let me convert it. Thanks